scripts/cassandra-integration-test.sh

```diff
@@ -26,6 +26,19 @@ setup_cassandra() {
   docker compose -f "$compose_file" up -d
 }
 
+healthcheck_cassandra() {
+  local cas_version=$1
+  local container_name="cassandra-${cas_version}"
+  local status="unhealthy"
+  while [[ ${status} != "healthy" ]]; do
```

Comment:
This is not how we usually perform liveness checks. This will loop forever, but we want to timeout at some point. Please check other scripts for deadline based checks.

Reply:
Done. Thanks